readme.txt

```diff
@@ -9,16 +9,41 @@ Smaily newsletter subscription plugin for WordPress
 
 == Description ==
 
-Smaily newsletter subscription plugin for WordPress.
+Smaily email marketing and automation extension plugin for Wordpress.
+Automatically generate a simple sign-up form and add new subscribers to Smaily subscribers list.
 
-== Installation ==
+= Features =
 
-* Install as regular plugin.
+**Wordpress Newsletter Subscribers**
+
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+**Subscription Widget**
+- Option for custom advanced HTML form.
+- Functional even with NoScript users.
+- Supports sending custom form values to Smaily.
+- Possible to add widget anywhere.
```

Comment:
You have "Possible to add widget anywhere." here, but on in README.md. Please add it there as well or remove it from readme.txt.